frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/storage-tab/vm-wizard-storage-modal-enhanced.tsx

```diff
@@ -95,10 +95,13 @@ const VMWizardStorageModal: React.FC<VMWizardStorageModalProps> = (props) => {
         usedDiskNames={usedDiskNames}
         usedPVCNames={usedPVCNames}
         templateValidations={templateValidations}
-        disk={diskWrapper}
-        volume={volumeWrapper}
-        dataVolume={dataVolumeWrapper}
-        persistentVolumeClaim={persistentVolumeClaimWrapper}
+        disk={new DiskWrapper(diskWrapper, true)}
+        volume={new VolumeWrapper(volumeWrapper, true)}
+        dataVolume={dataVolumeWrapper && new DataVolumeWrapper(dataVolumeWrapper, true)}
+        persistentVolumeClaim={
+          persistentVolumeClaimWrapper &&
```

Comment:
not really, because we need to preserve the information that the value is missing entirely and not just that the value is not filled (datavolume object exists).

I already tried that with empty objects and it was causing more problems than it solved

Reply:
This also creates an expectation that some wrappers might be falsy (undefined) so code working with them needs to keep that in mind.

Reply:
yes, it does